lib/src/data/authentication/firebase_authentication_repository.dart

```diff
@@ -1,15 +1,22 @@
 import 'dart:async';
 
 import 'package:allthenews/src/domain/authentication/authentication_repository.dart';
+import 'package:allthenews/src/domain/authentication/firebase_exception.dart';
```

Comment:
teraz zauważyłem, że plik się nie powinien tak nazywać (niespójnie względem klasy), jakbyś mógł przy okazji zmienić to byłoby super :)